pythia/modules/fusions.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+"""
+The fusions module contains various Fusions techniques some based on BLOCK:
+Bilinear Superdiagonal Fusion for VQA and VRD. For e.g. LinearSum, ConcatMLP
+taken from https://github.com/Cadene/block.bootstrap.pytorch#fusions.
+
+For implementing your own fusion technique, you need to follow these steps:
+
+.. code::
+    from pythia.common.registry import registry
+    from pythia.modules.fusions import BaseFusion
```

Comment:
We don't have a `BaseFusion` class anymore.
Can we list all of the Fusions available here?